doc/source/ray-air/examples/upload_to_wandb.ipynb

```diff
@@ -69,11 +68,8 @@
    "outputs": [],
    "source": [
     "def get_train_dataset() -> ray.data.Dataset:\n",
-    "    \"\"\"Return the \"Breast cancer\" dataset as a Ray dataset.\"\"\"\n",
-    "    data_raw = load_breast_cancer(as_frame=True)\n",
-    "    df = data_raw[\"data\"]\n",
-    "    df[\"target\"] = data_raw[\"target\"]\n",
-    "    return ray.data.from_pandas(df)"
+    "    dataset = ray.data.read_csv(\"s3://air-example-data/breast_cancer.csv\")\n",
```

Comment:
so -- does this work if you do NOT have any aws credentials configured?

Reply:
(please verify!)

Reply:
It does, I tried it on Colab:
<img width="1187" alt="Screenshot 2022-06-24 at 08 20 38" src="https://user-images.githubusercontent.com/10364161/175566427-dec6362d-55cf-4588-97f5-ef0c1e59a356.png">